plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/RemoteDockerNode.java

```diff
@@ -51,6 +52,10 @@
     private final String                         nodeHost;
     private final String                         nodeIp;
 
+    private volatile boolean isBound;
```

Comment:
i've just changed the logic to use a single flag, so keep calm, nothing to argue about